deep_qa/data/dataset.py

```diff
@@ -195,9 +165,22 @@ class IndexedDataset(Dataset):
     def __init__(self, instances: List[IndexedInstance]):
         super(IndexedDataset, self).__init__(instances)
 
+    def sort_by_padding(self, sorting_keys: List[str]):
+        """
+        Sorts the ``Instances`` in this ``Dataset`` by their padding lengths, using the keys in
```

Comment:
Can you add: "in the order that the sorting keys are passed" to the docs here, or a comment where you sort with the key as a list?